integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/ElasticsearchExtensionIT.java

```diff
@@ -167,6 +174,10 @@ public void query() {
 				.hasDocRefHitsAnyOrder( INDEX_NAME, FIRST_ID, SECOND_ID, THIRD_ID, FOURTH_ID, FIFTH_ID, EMPTY_ID )
 				.hasTotalHitCount( 6 );
 
+		Assertions.assertThat( result.getResponseBody() )
+				.isNotNull()
+				.extracting( body -> body.get( "_shards" ) ).isInstanceOf( JsonObject.class );
```

Comment:
Maybe I miss something, but it seems that we already test it in the method `query_gsonResponseBody`.

Reply:
Right, I made a mistake during a rebase